### DISCUSSION TITLE
Pimcore version 6.9 new changes

### DISCUSSION THREAD

**Discussion by Thejaswini-Rao-U** [pimcore/pimcore#15545] [Q&A / Support]
Hi 

Please can anyone share me the changes of class definition or functionality when we migrate from pimcore version 6.6 to 6.9.6 ?

i have migrated my system to pimcore 6.6 to 6.9.6 but i want to know class definition changes which is new in 6.9.6 and also know will it effect the current functionality?

Please help me

**Comment by kingjia90** [comment]
Have you already checked this page with all the upgrade notes from 6.6 to 6.9?
https://github.com/pimcore/pimcore/tree/6.9/doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes

Usually there are no class definition changes that affect the current functionality, because that's a BC break and can only be done in a major release

**Comment by Thejaswini-Rao-U** [comment]
@kingjia90 , Yes i have checked this document before upgrade, but i got few of the changes in the class definitions, which is not mentioned in pimcore document

Below few listed changes got in class definition

1. 'allowVariants' => NULL, changed to 'allowVariants' => false,

2. below 2 fields are removed from class definition in input & numeric field
'queryColumnType' => 'varchar',
  'columnType' => 'varchar',

3. 'blockedVarsForExport' => 
+                array (
+                ), =>  extra added in numeric field


4. 'generateTypeDeclarations' => false,
+   'showFieldLookup' => false,


5. queryColumnType' => 'text', in tag field
etc 
Hope these configuration fields will not get effected to current functionality. What you suggest?

**Comment by Thejaswini-Rao-U** [comment]
Fixed
